python/sglang/srt/layers/linear.py

```diff
@@ -1301,7 +1301,7 @@ def weight_loader_v2(self, param: BasevLLMParameter, loaded_weight: torch.Tensor
             # It does not support additional parameters.
             param.load_row_parallel_weight(loaded_weight)
 
-    def forward(self, input_, skip_all_reduce=False):
+    def forward(self, input_, skip_all_reduce=False, disable_symmetric_memory=True):
```

Comment:
Can we just check `not is_max_padding()` here instead of adding an arg?

Reply:
Good catch. Let me check.